manifest.json

```diff
@@ -11,7 +11,15 @@
   "content_scripts": [
     {
       "matches": ["*://*.lectio.dk/*"],
-      "js": ["colors.js", "assignment.js", "misc.js"]
+      "js": ["colors.js", "misc.js"]
+    },
+    {
+      "matches": ["*://*.lectio.dk/*/SkemaNy.aspx"],
```

Comment:
Does this work, if a get argument is applied? And do they ever apply a get argument?